### ISSUE TITLE
Fatal script error

### ISSUE THREAD

**Issue by lumor367** [spesmilo/electrum#6601]
Message error: "Failed to execute script run_electrum" . I've already googled all the options to fix this issue. I've upgraded the electrum for the last one 4.03. My windows 10 is also updated in last version as also the Python 3.8 till flashes, codecs, drivers, my pc hardwares, everything is updated, and restarted my pc several times. As also tried Electrum previous versions but still not opening it
![elecerror](https://user-images.githubusercontent.com/71442699/93468732-ce344800-f8c5-11ea-9ce9-9f8cee634a59.jpg)

. Even by python it just doesnt work.

**Comment by SomberNight** [comment]
Could you try running Electrum in a console/terminal, so that it can print a detailed error?

Unfortunately, on Windows, Command Prompt (cmd) does not work for this, as Electrum is compiled as a "windowed" application instead of a "console" application, so you would need to install a more powerful console.

Specifically, MinTTY would work. See instructions here: https://github.com/spesmilo/electrum/issues/2592#issuecomment-484562196

**Comment by SomberNight** [comment]
Alternatively, I could create a custom build of Electrum for you that is built as a "console" application, if you are willing to run that.

**Comment by lumor367** [comment]
> Could you try running Electrum in a console/terminal, so that it can print a detailed error?
> 
> Unfortunately, on Windows, Command Prompt (cmd) does not work for this, as Electrum is compiled as a "windowed" application instead of a "console" application, so you would need to install a more powerful console.
> 
> Specifically, MinTTY would work. See instructions here: [#2592 (comment)](https://github.com/spesmilo/electrum/issues/2592#issuecomment-484562196)

Yeah, I tried run in Powershell, in first moment I could to install the requirements such as symmetric ciphers pycryptodomex, but Im having problem with libsecp256k1. The install of libsecp256k1 occurs very well, but when I try to ./run_electrum then just shows this : 
"  \libsecp256k1-0.dll' (or one of its dependencies). Try using the full path with constructor syntax."), FileNotFoundError("Could not find module 'libsecp256k1-0.dll' (or one of its dependencies). Try using the full path with constructor syntax.")]
Error: Failed to load libsecp256k1.

I'm goin to try the Git

**Comment by SomberNight** [comment]
Note that what I am suggesting is not that you run Electrum from source and install dependencies yourself,
but rather that you use a "MinTTY" terminal to run the same Electrum binary you have already downloaded.

**Comment by lumor367** [comment]
> Note that what I am suggesting is not that you run Electrum from source and install dependencies yourself,
> but rather that you use a "MinTTY" terminal to run the same Electrum binary you have already downloaded.

Finally it opened: I've noticed that some dll files didnt into electrum subfolder as they are into Electrum main setup folder. I just copied all dll files and pasted into subfolder electrum, an then ./run. Finally works.

**Comment by SomberNight** [comment]
So are you running from source then? How exactly did you fix it?

Have you tried running the official binary with MinTTY? Do you have logs from that?

**Comment by lumor367** [comment]
> So are you running from source then? How exactly did you fix it?
> 
> Have you tried running the official binary with MinTTY? Do you have logs from that?

As I said before, I just copied all dll files from main setup fodlder into subfolder electrum then ./run_electrum in powershell

**Comment by SomberNight** [comment]
I am just having a hard time understand where you got the DLL files from, and how come you have a run_electrum script.
Specifically, you would only have a run_electrum script if you were running from source; and re the DLL files if you built them yourself they would already get placed in the expected location. That leaves me to believe that you are running from source but you copied the DLL files from the official release binary (presumably the `setup`/installer exe).

**Comment by lumor367** [comment]
Thats it!!! : "> That leaves me to believe that you are running from source but you copied the DLL files from the official release binary (presumably the `setup`/installer exe)."
